scripts/validate_docstrings.py

```diff
@@ -624,6 +625,10 @@ def get_validation_data(doc):
         errs.append(error('GL07',
                           correct_sections=', '.join(correct_order)))
 
+    if (doc.deprecated and not doc.name.startswith('pandas.Panel')
```

Comment:
Are there many wrong cases in `Panel`? if there are just few, I'd prefer to also fix them than hardcoding this here.

Reply:
https://github.com/pandas-dev/pandas/blob/77278ba8158457f142e6bba19e6d53a66a3f1322/scripts/validate_docstrings.py#L494-L499

Actually `Panel` has no wrong cases - I was removing it from checks due to this above existing code, which sets all `Panel` related objs as deprecated.

I have changed `validate_docstrings.py` now so that it checks these as well. Please have a look,